management-account/terraform/s3.tf

```diff
@@ -423,6 +423,71 @@ data "aws_iam_policy_document" "focus_reports_s3_bucket" {
   }
 }
 
+data "aws_iam_policy_document" "cur_reports_v2_hourly_ap_poc_s3_policy" {
+  version = "2012-10-17"
+
+  statement {
+    effect = "Allow"
+    actions = [
+      "s3:GetBucketPolicy",
+      "s3:GetBucketAcl"
+    ]
+    resources = ["arn:aws:s3:::moj-cur-reports-v2-hourly-replication-to-ap-poc"]
+
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::386209384616:root"]
```

Comment:
I understand this to be an AWS account number, can you link to any documentation for BCM where this is referenced?